tests/functional/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ set(OLP_SDK_FUNCTIONAL_TESTS_SOURCES
     ./olp-cpp-sdk-core/OlpClientDefaultAsyncHttpTest.cpp
     ./olp-cpp-sdk-dataservice-read/ApiTest.cpp
     ./olp-cpp-sdk-dataservice-read/CatalogClientTest.cpp
+    ./olp-cpp-sdk-dataservice-read/DataserviceReadVersionedLayerClientTest.cpp
```

Comment:
Why is it called DataserviceReadVersionedLayerClientTest.cpp?
It is already under the dataservice-read path so no need to call it extra DataServiceRead...

Reply:
There was a specific requirement to name the file like this already - to align the test class name.